src/components/FormAddMovie/index.jsx

```diff
@@ -5,10 +5,10 @@ import Select from "react-select";
 import { Formik, Field, Form, ErrorMessage } from "formik";
 import movieService from "../../services/movie.service";
 import * as Yup from "yup";
-import { useSnackbar } from "react-simple-snackbar";
+// import { useSnackbar } from "react-simple-snackbar";
 
 export const FormAddMovie = ({ movie, onCreateMovieSuccess }) => {
-  const [openSnackbar, closeSnackbar] = useSnackbar();
+  // const [openSnackbar, closeSnackbar] = useSnackbar();
```

Comment:
PLease mock instead of comment:

`
import * as Snackbar from 'react-simple-snackbar'

describe("Card", () => {

// Mocks the open and close functions
  const openSnackbarMock = jest.fn()
  const closeSnackbarMock = jest.fn()
  jest.spyOn(Snackbar, 'useSnackbar').mockImplementation(() => [openSnackbarMock, closeSnackbarMock])


  it("renders appropriately", () => {
    render(<MemoryRouter><Header /></MemoryRouter>);
    expect(screen.findAllByDisplayValue("FIND YOUR MOVIE"));
  });
});

`

find more information in the next link https://evandromacedo.github.io/react-simple-snackbar/